projects/rero/ng-core/src/lib/record/record.service.ts

```diff
@@ -218,21 +223,18 @@ export class RecordService {
   }
 
   /**
-   * Create a new record
+   * Update a record
    * @param recordType - string, type of resource
-   * @param record - object, record to create
    * @param pid - string, record PID
+   * @param record - object, record to update
    */
-  update(recordType: string, record: { pid: string }) {
-    const url = `${this._apiService.getEndpointByType(recordType, true)}/${
-      record.pid
-      }`;
-    return this._http
-      .put(url, record)
-      .pipe(
-        catchError((error) => this._handleError(error)),
-        tap(() => this.onUpdate.next(this._createEvent(recordType, { record })))
-      );
+  update(recordType: string, pid: string, record: any) {
```

Comment:
I do not understand why separate the pid and the record is required. This is a breaking change.